jdbc/core/src/main/java/org/apache/shardingsphere/driver/jdbc/core/driver/ShardingSphereDriverURL.java

```diff
@@ -57,7 +62,25 @@ public ShardingSphereDriverURL(final String url) {
     @SneakyThrows(IOException.class)
     public byte[] toConfigurationBytes() {
         try (InputStream stream = inClasspath ? ShardingSphereDriverURL.class.getResourceAsStream("/" + file) : Files.newInputStream(new File(file).toPath())) {
-            byte[] result = new byte[null == stream ? 0 : stream.available()];
+            LineProcessor<byte[]> lineProcessor = new LineProcessor<byte[]>() {
+                
+                private final StringBuilder builder = new StringBuilder();
+                
```

Comment:
Could we simplify it with `Files.readAllLines`? Refer to `DataSourcePersistServiceTest.readDataSourceYaml` for more details.

And guava dependency might be removed later.

Reply:
File reading will have problems when packaged into jar